src/viser/_messages.py

```diff
@@ -515,34 +549,15 @@ class GuiRemoveMessage(Message):
 
 @dataclasses.dataclass
 class GuiUpdateMessage(Message):
-    """Sent client->server when a GUI input is changed."""
+    """Sent client<->server when any property of a GUI component is changed."""
 
     id: str
-    value: Any
-
-
-@dataclasses.dataclass
-class GuiSetVisibleMessage(Message):
-    """Sent client->server when a GUI input is changed."""
+    prop_name: str
+    prop_value: Any
```

Comment:
FYI @jkulhanek the main change I've made so far is to simplify this `GuiUpdateMessage`; the types in yours were stronger for the client but I didn't feel like the complexity tradeoff was worth it

Reply:
(please let me know if you disagree with this! or if there's something I missed)

Reply:
I don’t mind, both are fine I think. The downside of the newer message is that you can change only a single property at a time. The situation where this might be useful is eg when you change the options of a dropdown and you want to change the value as well.

Reply:
If you can only change a single property at the time, you have to put the dropdown temporarily into invalid state.

Reply:
Yeah, makes sense. In this case I guess we can just send two messages, when they're sent `server -> client` the messages are windowed/batched before being sent anyways so the performance penalty should be small.

Reply:
> you have to put the dropdown temporarily into invalid state

Oops race condition with my message, this is a valid concern. In practice because of the windowing/batching mechanisms I don't think this will be an issue, but it doesn't feel great... will think about it.

Reply:
Yeah, we can also switch to `Dict[str, Any]`, which is similar to before. A lot of the complexity was in the kwargs handling, custom serialization, more advanced TypeScript stuff, etc, and not in intrinsic to the move to single-prop updates.